extensions-contrib/rocketmq-indexing-service/src/main/java/org/apache/druid/indexing/rocketmq/RocketMQConsumerConfigs.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.rocketmq;
+
+import org.apache.druid.common.utils.IdUtils;
+import org.apache.druid.java.util.common.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Common place to keep all rocketmq consumer configs
+ */
+public class RocketMQConsumerConfigs
```

Comment:
I guess this class follows the `KafkaConsumerConfigs` class,  but that class is shared at least twice if I remember correctly. So I think there's a strong need of this class and we could move the code of consumer group generation to where instance of RocketMQ  consumer is created.

Reply:
Deleted.